diff --git a/node_modules/vuepress-theme-gungnir/lib/client/components/Navbar.vue b/node_modules/vuepress-theme-gungnir/lib/client/components/Navbar.vue
index 387ea01..2e8a6b8 100644
--- a/node_modules/vuepress-theme-gungnir/lib/client/components/Navbar.vue
+++ b/node_modules/vuepress-theme-gungnir/lib/client/components/Navbar.vue
@@ -59,7 +59,7 @@ const router = useRouter();
 const navbar = ref<HTMLElement | null>(null);
 const siteBrand = ref<HTMLElement | null>(null);
 const siteBrandLink = computed(() => themeLocale.value.home || "/");
-const siteBrandTitle = "$ cd /home/";
+const siteBrandTitle = computed(() => themeLocale.value.homeText ?? "$ cd /home/");
 const linksWrapperMaxWidth = ref(0);
 const linksWrapperStyle = computed(() => {
   if (!linksWrapperMaxWidth.value) {
